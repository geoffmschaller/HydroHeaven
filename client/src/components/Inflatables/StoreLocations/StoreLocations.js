import React from 'react';
import styles from './StoreLocations.module.css';
import AG_LOCATION_IMAGE from '../../../static/images/locations/arroyo_grande.jpg';
import SLO_LOCATION_IMAGE from '../../../static/images/locations/san_luis_obispo.jpg';

const StoreLocations = () => (
	<div className={styles.storeLocations}>
		<div className={styles.location}>
			<img src={AG_LOCATION_IMAGE} alt="" />
			<div className={styles.address}>135 Traffic Way - Arroyo Grande, CA</div>
			<div className={styles.phone}>(805) 473 - 2205</div>
		</div>
		<div className={styles.location}>
			<img src={SLO_LOCATION_IMAGE} alt="" />
			<div className={styles.address}>200 Higuera Street - San Luis Obispo, CA</div>
			<div className={styles.phone}>(805) 544 - 1772</div>
		</div>
		<div className={styles.service}>
			<div className={styles.serviceHolder}>
				<div className={styles.title}>Hours of Operation:</div>
				<div className={styles.day}>Monday - Friday: 9:30A - 5:30P</div>
				<div className={styles.day}>Saturday: 10A - 5P</div>
				<div className={styles.day}>Sunday: CLOSED</div>
			</div>
		</div>
	</div>
);

export default StoreLocations;
