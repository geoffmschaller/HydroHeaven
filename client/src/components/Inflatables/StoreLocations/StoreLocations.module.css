.storeLocations {
  width: 100%; }
  .storeLocations .location {
    width: 100%;
    margin-bottom: 25px; }
    .storeLocations .location img {
      width: 100%; }
    .storeLocations .location .address {
      font-size: 15px;
      color: #808b96; }
    .storeLocations .location .phone {
      font-size: 18px;
      color: #566573; }
