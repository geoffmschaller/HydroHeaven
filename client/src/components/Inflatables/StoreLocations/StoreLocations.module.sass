$address-color: #808b96
$phone-color: #566573

.storeLocations
	width: 100%

	.location
		width: 100%
		margin-bottom: 25px

		img
			width: 100%

		.address
			font-size: 15px
			color: $address-color

		.phone
			font-size: 18px
			color: $phone-color