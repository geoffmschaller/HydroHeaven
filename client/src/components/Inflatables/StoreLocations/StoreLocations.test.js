import React from 'react';
import StoreLocations from "./StoreLocations";

const wrapper = <StoreLocations/>;

it("Store Locations Inflatable", () => {
	expect(wrapper).toMatchSnapshot();
});