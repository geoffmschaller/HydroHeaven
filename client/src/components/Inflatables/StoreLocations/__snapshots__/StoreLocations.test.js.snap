// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Store Locations Inflatable 1`] = `<StoreLocations />`;
